src/mpid/ch4/netmod/ofi/ofi_pre.h

```diff
@@ -171,7 +171,9 @@ typedef struct {
 
 typedef struct {
     fi_addr_t dest;
-#if defined MPIDI_OFI_ENABLE_SCALABLE_ENDPOINTS || defined MPIDI_OFI_ENABLE_RUNTIME_CHECKS
+#if MPIDI_OFI_ENABLE_RUNTIME_CHECKS
+    unsigned ep_idx:MPIDI_OFI_MAX_ENDPOINTS_BITS_SCALABLE;
+#elif MPIDI_OFI_ENABLE_SCALABLE_ENDPOINTS
```

Comment:
@pkcoff pointed out that this causes problems on [src/mpid/ch4/netmod/ofi/ofi_types.h#L187](https://github.com/pmodels/mpich/blob/9ea3b1c104b486b2d4c40bf3697cdaf56c9484a3/src/mpid/ch4/netmod/ofi/ofi_types.h#L187) too. Let me think about if there's a way we can improve this.

This only worked in the past because the original check always evaluated to true.